### ISSUE TITLE
Discussion: Disable Trimming Whitespace when Working on .md files in `.editorconfig?

### ISSUE THREAD

**Issue by Sewer56** [Nexus-Mods/NexusMods.App.Meta#11]
## Note

Putting this up as a discussion topic for the interested.

## Current Behaviour / Context

When whitespace trimming is enabled, creating line breaks with `  ` in markdown is not possible.

For use within `MkDocs`, and various other Markdown renderers, you might need to put double space `  ` at the end of a line to ensure the next bullet ends up in a new line. This can be desirable to optimize for text layout.

However currently, those doublespaces will be stripped under our current config.

## Alternatives

Alternatives include `\` (supported in some Markdown renderers) and using raw HTML `<br/>` to do the job.

MkDocs supports the latter, but not the former.

## Configuration Change

```
[*.md]
trim_trailing_whitespace = false
```

**Comment by erri120** [comment]
Trailing whitespace is an anti-pattern in most markup languages. If I'm understanding your issue correctly, double line-breaks should do the trick as well.

**Comment by Sewer56** [comment]
This was not a recommendation.
This was a discussion topic (note: we have discussions disabled).

I was asking on people's opinions regarding the matter. 

While I personally consider this an anti pattern too, I found that at one point I had to disable this explicitly (even if temporarily) to get something done (more specifically, I wanted to avoid bullet point formatting). 

Since I did this, I figured there was a good enough use case that needed me to do so, at the time.
Of course I found a workaround since, but I figured I'd just want to hear what people think about it.
